regolith/helpers/l_milestoneshelper.py

```diff
@@ -160,10 +160,12 @@ def sout(self):
             rc.stati = PROJECTUM_STATI
         for projectum in collection:
             projectum["deliverable"].update({"name": "deliverable",
-                                             "objective": "deliver"})
+                                             "objective": "deliver",
+                                             "uuid": projectum.get('_id')})
             milestones = [projectum["deliverable"]]
             if projectum.get("kickoff"):
-                projectum["kickoff"].update({"type": "meeting"})
+                projectum["kickoff"].update({"type": "meeting",
+                                             "uuid": "ko" + projectum.get('_id')})
```

Comment:
use an f-string here `f"ko{projectum.get('_id')}"` (and always for composing strings)

Reply:
Will do.